deps/build.jl

```diff
@@ -59,6 +59,7 @@ end
 
 # Windows
 if is_windows()
+    rm(joinpath(dirname(@__FILE__), "usr"); force = true, recursive = true)
```

Comment:
This deletes the binaries every time, and will cause a lot of unnecessary re-downloading. Files shouldn't be deleted automatically, instead we can provide a warning to the user that the validation of the existing files failed and what command they should run to clean the binaries.